src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -82,112 +82,6 @@ def chunk_iter(inputs, feature_extractor, chunk_len, stride_left, stride_right,
             break
 
 
-def _find_timestamp_sequence(sequences, tokenizer, feature_extractor, max_source_positions):
```

Comment:
It doesnt' handle, and cannot handle language though.

This is because this function returns a pure sequence of tokens, which still needs to be decoded specially by the tokenizer.

The new function handles properly the 4 arguments (return_timestamps:true/false, return_language:true/false) while this handles only 1 (and there is still complexity within `decode` which should be accounted for.

There's a reason I moved everything around, and as explicited in the PR, it's because I couldn't find any other way which could keep the current architecture. It's not about this particular piece of code.